src/ListToggleCommand.js

```diff
@@ -49,7 +50,19 @@ export class ListToggleCommand extends UICommand {
     if (!nodeType) {
       return tr;
     }
-
+    // [FS] IRAD-1391 2021-05-25
+    // For Citation applied paragraph list is not applying
+    // the issue is because of position and node type
+    let selectedNode = tr.doc.nodeAt(selection.from);
+    if (null == selectedNode) {
+      let pos = selection.from - 2;
+      pos = pos < 0 ? 0 : pos;
+      selectedNode = tr.doc.nodeAt(pos);
+      if (selectedNode && 'citationnote' === selectedNode.type.name) {
```

Comment:
This is the wrong place for this code.  You're adding custom code to the editor to fix bugs created by a plugin.   The fix should be in the plugin